src/firebase/firestore/job.ts

```diff
@@ -79,17 +79,24 @@ export const getAllJobs = async (): Promise<Job[]> => {
       promises.push(parseJob(job));
     });
     const allJobs = await Promise.all(promises);
-    return allJobs;
+    return allJobs.filter((obj) => obj.hasOwnProperty('jobPosition')); // filter out metadata and anything w/o required description;
   } catch (e) {
     console.error(e);
     throw e;
   }
 };
 
-// export const updateJob = async (job_id: string): Promise<void> => {};
-export const deleteJob = async (job_id: string): Promise<void> => {
+export const updateJob = async (jobId: string, newLiked: boolean): Promise<void> => {
```

Comment:
maybe a bit more descriptive name since this is just for updating the liked boolean for a job

Reply:
fixed